libkpod/stats.go

```diff
@@ -0,0 +1,161 @@
+package libkpod
+
+import (
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer"
+)
+
+// ContainerStats contains the statistics information for a running container
+type ContainerStats struct {
+	Container   string
+	CPU         float64
+	cpuNano     uint64
+	systemNano  uint64
+	MemUsage    uint64
+	MemLimit    uint64
+	MemPerc     float64
+	NetInput    uint64
+	NetOutput   uint64
+	BlockInput  uint64
+	BlockOutput uint64
+	PIDs        uint64
+}
+
+// GetContainerStatsFollow gets the running stats for a running container continually
+func (c *ContainerServer) GetContainerStatsFollow(ids []string, all bool, statsChan chan []*ContainerStats, times int) error {
```

Comment:
I'd generally prefer to keep API functions acting on a single container - accept a single ID and channel, let the kpod command call this multiple times